test/e2e/live/cua-gpu-qualification.test.ts

```
@@ -0,0 +1,1670 @@
+            "--json",
+          ],
+          {
+            artifactName: "cleanup-cua-qualification-target",
+            captureLimitBytes: CUA_QUALIFICATION_FILE_MAX_BYTES,
+            env: runtimeEnv,
+            redactionValues,
+            timeoutMs: 90_000,
+          },
+        );
+        if (result.exitCode !== 0) {
+          throw new Error(`CUA qualification target cleanup failed: ${result.stderr}`);
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)